src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,15 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash:
+      indent_size: true
```

Comment:
Well, the option appears in the Atom settings but I don't see'm in the file options.md Oo ? 

![2016_09_29_01](https://cloud.githubusercontent.com/assets/5594303/18964750/8316d0ac-867a-11e6-9cb2-464fdfa13ece.png)